addons/cookoff/functions/fnc_cookOffEffect.sqf

```diff
@@ -183,5 +184,16 @@ if (isServer) then {
             [2 + random 1], 1, 0, "", "", _obj
         ];
     };
-}, 0, [_obj, _jet, _ring, _time, CBA_missionTime, _light, _fireSelection, _sound]] call cba_fnc_addPerFrameHandler;
+    
+    private _tiPars = getVehicleTIPars _obj; 
+    [QGVAR(setTIPars), [
+        _obj, [
+            ((_tiPars select 0) + (_intensity * 0.01))/1.005, 
+            ((_tiPars select 1) + (_intensity * 0.004))/1.002, 
+            ((_tiPars select 2) + (_intensity * 0.01))/1.005
+        ]
+    ]] call CBA_fnc_globalEvent;
```

Comment:
This raises a global events from globally executed script I think.

Reply:
Yeah, this should be called locally now. This function is run per client